.github/workflows/code-quality.yml

```diff
@@ -100,6 +100,12 @@ jobs:
           clang-format --version
           cpplint --version
 
+      - name: CMake lint
+        shell: bash
+        run: |
+          python -m pip install cmakelang pyyaml
+          find . \( -name "CMakeLists.txt" -o -wholename "*/cmake/*.cmake" \) | xargs cmake-lint
```

Comment:
[consider][minor] I had to look up `-wholename`, and `man` said `-path` is more portable

>        -wholename pattern
>              See -path.  This alternative is less portable than -path.

It works on CI so all good, but if someone e.g. on MacOS tried to lift this command to run locally, it might not work.

Reply:
Good spot, replaced with path https://github.com/OpenAssetIO/OpenAssetIO/pull/811/commits/c37e253dcc71eb7d0c1f9c5c67b6a2002b21aa5e

Reply:
Just for fun, `find . -not -path '*/.*' \( -name "CMakeLists.txt" -o -name "*.cmake" \)` seems to produce the same result.